dockstore-webservice/src/main/java/io/dockstore/webservice/languages/WDLHandler.java

```diff
@@ -226,6 +226,16 @@ public VersionTypeValidation validateEntrySet(Set<SourceFile> sourcefiles, Strin
                 String content = FileUtils.readFileToString(tempMainDescriptor, StandardCharsets.UTF_8);
                 checkForRecursiveHTTPImports(content, new HashSet<>());
 
+                // Check for local recursive imports in the WDL files
```

Comment:
This code mostly duplicates similary code in parseWorkflowContent. Could you create a new function and call from both places?

Reply:
Other thought: If this is similar to parseWorkflowContent, does it mean we end up looking through all the imports twice?
Is there a way to avoid this?

Reply:
Other thought: If this is similar to parseWorkflowContent, does it mean we end up looking through all the imports twice?
Is there a way to avoid this?

Reply:
added a new function